### DISCUSSION TITLE
ALTER PUBLICATION fails SQL syntax check in dashboard but is fine in `psql`

### DISCUSSION THREAD

**Discussion by tvogel** [supabase/supabase#16935] [Contribute to Supabase]
# Bug report

## Describe the bug
`ALTER PUBLICATION` [syntax](https://www.postgresql.org/docs/15/sql-alterpublication.html) not fully supported in dashboard.

## To Reproduce

Trying to run
```SQL
ALTER PUBLICATION supabase_realtime ADD TABLE auth.users (id, raw_app_meta_data);
```
in the dashboard, gives me a `Failed to validate sql query: syntax error at or near "("` error. 

## Expected behavior
However, in `psql` I get success:
```SQL
postgres=> ALTER PUBLICATION supabase_realtime ADD TABLE auth.users (id, raw_app_meta_data);
ALTER PUBLICATION
```

## Context
This probably also applies to `CREATE PUBLICATION` and variants like `ALTER PUBLICATION ... SET TABLE`.

## System information

- OS: openSUSE Tumbleweed
- Version of supabase-js: n/a
- Version of Node.js: n/a

**Comment by GaryAustin1** [comment]
This is probably related to Postgres 15 features.  There was some other new 15 feature that also does not work in the SQL editor because of syntax reported.

Supabase needs to update the syntax checker for 15 features.
